ui/app/components/string-list.js

```diff
@@ -62,6 +62,7 @@ export default Component.extend({
     *
     */
   inputList: computed(function() {
+    /* eslint-disable ember/no-side-effects */
```

Comment:
what side effect are we allowing here?

Reply:
Here's the rule: https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-side-effects.md And looking at this now, I think we should change this to happen on init, but since this is a backport I didn't want to do it here. I'll make a follow up PR!

Reply:
But why do we need it?

Reply:
Oh - linting was failing on this - looks like the ignore was added at some point in master after 1.1.3.